packages/xo-server/src/xo-mixins/pool.js

```diff
@@ -81,4 +81,35 @@ export default class Pools {
       await _xo.mergeXenPools(source._xapiId, target._xapiId, force)
     }
   }
+
+  async listPoolsMatchingCriteria({ minCpus = 0, minMemory = 0, minSrSize = 0, poolTag, srTag }) {
```

Comment:
Why `poolTag`?

You did not named other params `minPoolCpus`, etc.

Reply:
I misunderstood the client's request, so I added the tag check.

Now, I check the names of pools and SRs instead of tags.